assets/js/pages/ClusterDetails/ClusterDetailsPage.test.jsx

```diff
@@ -10,13 +10,16 @@ import { ClusterDetailsPage } from './ClusterDetailsPage';
 
 describe('ClusterDetails ClusterDetailsPage component', () => {
   it.each([
-    { type: 'hana_scale_up', label: 'HANA Scale Up' },
+    { type: 'hana_scale_up', label: 'HANA Scale Up', scneario: 'unknown' },
```

Comment:
Typo in `scneario`.
Anyway, this test looks really inconsistent. You shouldn't put the scenario only in one of the entries, and you should actually update the `label` as you are testing that, the label with different hana scenario, right?